switch/mupdf/mupdf.patch

```diff
@@ -0,0 +1,66 @@
+diff -burN mupdf-1.16.1-source.orig/source/fitz/stream-open.c mupdf-1.16.1-source/source/fitz/stream-open.c
+--- mupdf-1.16.1-source.orig/source/fitz/stream-open.c	2019-08-02 09:49:26.000000000 -0500
++++ mupdf-1.16.1-source/source/fitz/stream-open.c	2019-10-01 00:31:31.325868186 -0500
+@@ -101,12 +101,21 @@
+ {
+ 	FILE *file;
+ 	unsigned char buffer[4096];
++	const char *file_name;
+ } fz_file_stream;
+ 
+ static int next_file(fz_context *ctx, fz_stream *stm, size_t n)
+ {
+ 	fz_file_stream *state = stm->state;
+ 
++	/* FIXME: Multiple calls to fseek/fread return incorrect part of file */
```

Comment:
This should be fixed now I hope. We corrected an issue with file buffering with devkitA64 r15